src/wp-settings.php

```diff
@@ -389,6 +389,11 @@
 require ABSPATH . WPINC . '/interactivity-api/class-wp-interactivity-api-directives-processor.php';
 require ABSPATH . WPINC . '/interactivity-api/interactivity-api.php';
 
+// Register the default theme directory root.
+// This has to happen before wp_script_modules is registered
+// because wp_script_modules needs to know if the current theme is a block theme.
+register_theme_directory( get_theme_root() );
```

Comment:
Should we rather move `wp_script_modules()->add_hooks();` to be later?

Reply:
Well what you have now here is a bigger risk because it registers the theme directory waaaaay earlier, even before mu-plugins are loaded. So I wouldn't wanna do that.

`wp_script_modules` on the other hand is a brand new feature in 6.5, there can't be any regression because it didn't exist before.

Reply:
Makes sense :)

Reply:
Done!